web/thesauruses/c/17/functions.json

```diff
@@ -0,0 +1,90 @@
+{
+    "meta": {
+        "language": "c",
+        "language_version": "17",
+        "language_name": "C",
+        "structure": "functions"
+    },
+    "concepts": {
+        "void_function_no_parameters": {
+            "code": "void func(){return;}",
+            "comment": "Return statement is optional.",
+            "name": "Function that does not return a value and takes no parameters"
+        },
+        "void_function_with_parameters": {
+            "code": "void foo(char c, int i, double d) { return; }",
+            "comment": "Return statement is optional.",
+            "name": "Function that does not return a value and that takes 1 or more defined parameters"
+        },
+        "void_function_with_keyword_parameters": {
+            "code": "void sum(int a=5,int b=7)",
+            "comment":"default paramaters",
+            "name": "Function that does not return a value and that takes 0 or more defined keyword parameters"
+        },
+        "void_function_variable_parameters": {
+            "code": "eg : void avg(int count,...)",
```

Comment:
`eg :` isn't valid code in C, so that needs to be removed. Only code that could be copied/pasted should be put in a "code" line. (Same for everywhere else you use `eg` below.)